_posts/2018-02-12-deep-learning-popularity.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: Why are deep learning models so popular?
+author: jessevuorinen
+excerpt: Thoughts on why deep learning models are popular, and some tips on how to get started with them.
+tags:
+- data science
+- neural network
+- supervised learning
+- artificial intelligence
+---
+
+Deep learning (i.e. big neural networks) is at the forefront of the ongoing trend of artificial intelligence and data science. Last year, a neural network based AI beat a human grandmaster in Go, one of the hardest board games. From machine translators to self-driving vehicles, it seems deep neural networks can be found in most state-of-the-art AI solutions. In our client projects, we have leveraged deep learning in image recognition problems and multivariate time series forecasting, for example. What makes it possible for neural networks to be able to generalize to such a broad range of problems? Why are they so useful?
```

Comment:
“What makes it possible…” is OK, but something like "Why are neural networks applicable to such a broad range of problems?" might be more succinct.